src/main/java/net/mcreator/ui/modgui/ModElementChangedListener.java

```diff
@@ -115,9 +112,13 @@ private boolean isGenericComponent(Component component) {
 	}
 
 	@Override default void actionPerformed(ActionEvent e) {
-		if (e.getSource() instanceof JComboBox) {
-			if (e.getModifiers() != 0)
+		if (e.getSource() instanceof JComboBox box) {

```

Comment:
`e.getModifiers() != 0` should likely be here given it is shared in both branches.

On that topic, modifiers also have constants defined so use that to make this more readable. A comment wouldn't hurt either.